awareness.js

```diff
@@ -156,6 +156,54 @@ export class Awareness extends Observable {
   }
 }
 
+/**
+ * A state encoder that serializes state into JSON.
+ *
+ * This is the default strategy used in this package.
+ */
+export class DefaultAwarenessStateEncoder {
+  /**
+   * Encode an awareness state entry
+   * @param {encoding.Encoder} encoder
+   * @param {any} update
+   */
+  static encodeState (encoder, update) {
```

Comment:
I've moved these to static.

A different pattern we could consider here is to instead instantiate a copy of this class and store it on an exported constant. I'm happy to change to that if you'd prefer.